cluster/local/README.md

```diff
@@ -1,29 +1,51 @@
 # Crossplane Local Deployment and Test
 
-The Local Framework is used to run end to end and integration tests on Crossplane. The framework depends on a running instance of Kubernetes.
-The framework also provides scripts for starting Kubernetes using `minikube` so users can
-quickly spin up a Kubernetes cluster.The Local framework is designed to install Crossplane, run tests, and uninstall Crossplane.
-
-## Requirements
-1. Docker version => 1.2 && < 17.0
-2. Ubuntu 16, Fedora 29 (the framework has only been tested on these versions of linux distributions)
-3. Kubernetes with kubectl configured
-4. Crossplane
+The Local Framework is used to run end to end and integration tests on Crossplane.
+The framework depends on a running instance of Kubernetes. The framework also
+provides scripts for starting a local Kubernetes using `kind`, `minikube` and `microk8s`
+so users can quickly spin up a Kubernetes cluster.The Local framework is designed
+to install Crossplane, run tests, and uninstall Crossplane.
 
 ## Instructions
 
 ## Setup
 
 ### Install Kubernetes
-You can choose any Kubernetes flavor of your choice.  The test framework only depends on kubectl being configured.
-The framework also provides scripts to install Kubernetes.
+You can choose any Kubernetes flavor of your choice.  The test framework only
+depends on kubectl being configured. The framework also provides scripts to install Kubernetes.
 
-- **Minikube** (recommended for MacOS): Run [minikube.sh](./minikube.sh) to setup a single-node Minikube Kubernetes.
+- **kind**: Run [kind.sh](./kind.sh) to setup a single-node Minikube Kubernetes.
+  - kind v0.8.0 and higher is supported
```

Comment:
Was about to comment on that we use `kind` v0.7.0 in `build`, but then I noticed that you are using whatever `kind` version the user decides to install for these cases, which I think makes sense for local dev scenarios. I also wonder if we could clean-up how we vendor tools in https://github.com/upbound/build/blob/e806fbf89e0c6e13cac0331f4cbdc1bf53ba06c6/makelib/k8s_tools.mk. I would like to move to pinning tools in the same way that we do `controller-gen` if possible. Anyway, that is a conversation for another day :sweat_smile:

Reply:
Yeah, theoretically all versions should be OK but for local dev 0.8.0 is much safer IMO thanks to not wiping out the cluster after restarts.